DEV: add warning shortcode to FT.CREATERULE command page (#418)
redis/docs: content/commands/ts.createrule/index.md

- If no samples are added to the source time series during a bucket period. no _compacted sample_ is added to the destination time series.
- The timestamp of a compacted sample added to the destination time series is set to the start timestamp the appropriate compaction bucket. For example, for a 10-minute compaction bucket with no alignment, the compacted samples timestamps are `x:00`, `x:10`, `x:20`, and so on.
- Deleting `destKey` will cause the compaction rule to be deleted as well.

{{% warning %}}

> On a clustered environment, [hash tags]({{< relref "/operate/oss_and_stack/reference/cluster-spec" >}}#hash-tags) should be used to force `sourceKey` and `destKey` to be stored in the same hash slot.

Reviewer: Maybe this? Avoids PV and makes it clear that you can't do it without hash tags.
```suggestion
On a clustered environment, you must use [hash tags]({{< relref "/operate/oss_and_stack/reference/cluster-spec" >}}#hash-tags) to force `sourceKey` and `destKey` to be stored in the same hash slot.
```

Author: "Must" might be too strong, but let's see.
